core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -793,6 +822,18 @@ private static Map<String, SnapshotRef> validateRefs(
     return inputRefs;
   }
 
+  private long computeStatisticsCheckpointSnapshotID() {
+    if (statisticsFilesById.isEmpty()) {
+      return -1;
+    }
+    for (Snapshot snapshot : Lists.reverse(snapshots)) {
```

Comment:
Please update this to conform to style guidelines. There should be empty newlines between control flow blocks and the following statements.